guacamole/src/main/java/org/apache/guacamole/tunnel/StandardTokenMap.java

```diff
@@ -73,7 +73,7 @@
      * The date format that should be used for the time token. This format must
      * be compatible with Java's SimpleDateFormat.
      */
-    private static final String TIME_FORMAT = "HHmmss";
+    private static final String TIME_FORMAT = "HHmmss-SSS";
```

Comment:
Why is the format being changed?

Reply:
I agree that this change need to be justified.

Assuming we do want such a token, it will probably need to be a different token. Modifying a well-established token to use a different format will result in breakage in downstream systems that rely on the format of the old token.